astropy/io/misc/asdf/tags/time/tests/test_timedelta.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+import pytest
+
+asdf = pytest.importorskip('asdf')
+from asdf.tests.helpers import assert_roundtrip_tree
+
+from astropy.time import Time
+
+
+@pytest.mark.parametrize('fmt', Time.FORMATS.keys())
```

Comment:
I think the tests should run over the much more limited set of `TimeDelta.FORMATS` - as is, all tests will generate a `TimeDelta` with `format='jd'.
For the new tests, you can avoid problems with types in the initialization by creating it in one format and then transforming to another:
```
td = TimeDelta(0.125, format='jd', scale='tai')
td = TimeDelta(td, format=fmt)
...
```